xtokens/src/lib.rs

```diff
@@ -467,10 +472,6 @@ pub mod module {
 			dest: MultiLocation,
 			dest_weight: Weight,
 		) -> DispatchResult {
-			ensure!(
```

Comment:
we still need this check

Reply:
I got your point, because `do_transfer_multiassets` is called by many places, so we cannot simply skip this check, right?.
If you think it's necessary, I will close the pr.

Reply:
yes. but I think we are fine to check it twice for the currencies?

Reply:
I rollbacked it. Check currencies twice.